### ISSUE TITLE
Minor "Error" in Tutorial

### ISSUE THREAD

**Issue by kiunthmo** [tensorflow/tensorflow#27418]
In the dristributed training tutorial (link below) the ordering of the lines can cause problems later on. If the lines are switched it will save some users headaches.

https://www.tensorflow.org/guide/distribute_strategy#using_tfdistributestrategy_with_custom_training_loops

```
with mirrored_strategy.scope():

  iterator_init = input_iterator.initialize()

  var_init = tf.global_variables_initializer()

  loss = train_step()

  with tf.Session() as sess:

    sess.run([iterator_init, var_init])

    for _ in range(10):

      print(sess.run(loss))
```

If you adapt the current tutorial code to use a model with weights it will currently not initialise the variables because of the ordering. I recommend switching the var_init and loss lines since the model is being built within the train_step().

**Comment by muddham** [comment]
@kiunthmo Thanks for bringing this to our notice. We will take a look and resolve it. Thanks!
